docs/golden-path/create-app/installing-plugins.md

```diff
@@ -0,0 +1,61 @@
+---
+id: installing-plugins
+sidebar_label: 003 - Installing plugins
+title: 003 - Installing plugins
+---
+
+Now that you have a working Backstage app, let's walk through the most valuable part of the Backstage ecosystem - plugins!
+
+## What is a Backstage plugin?
+
+A Backstage plugin usually consists of frontend and backend functionality. Some examples of Backstage plugins are our Software Catalog, Search, and Software Templates plugins! Each plugin provides a series of well-contained focused features, for example - the Software Catalog contains an entity ingestion engine, an optimized query layer for fetching entity information and a series of UI elements that provide list and detail functionality for entities. Some plugins allow modules which supplement existing plugin-level functionality, customizing it for specific use cases - a good example here are catalog processor modules which allow for ingesting data from common sources into the catalog.
+
+You'll generally have multiple packages that combine into a single "plugin". The common naming standard (detailed in [ADR-11](../../architecture-decisions/adr011-plugin-package-structure.md)) is demonstrated below for plugin `x`:
+
+> `x`: Primary frontend entrypoint, contains frontend-only code for the plugin.
```

Comment:
Is it important enough to add that when creating a new package for your plugin, you only need to specify the `id`? `-react`, `-node`, `-common` get appended for you.